src/vuex/types.spec.js

```diff
@@ -59,6 +83,12 @@ describe('vuex types unit tests', () => {
       ...getModuleKeys(identities),
       ...getModuleKeys(idleHandler),
       ...getModuleKeys(kycRecovery),
+      ...getModuleKeys(createAssetRequest),
```

Comment:
merge `createAssetRequest` and `updateAssetRequests` modules into the one `assetRequests` module